posts/five-interns-walk-into-an-office.md

```diff
@@ -0,0 +1,44 @@
+# Five interns walk into an office...
+This blog represents the combined experiences of the five interns (Clement Dessoude, Yashar Atajiang, Auguste Lehuger, Spencer Hamilton, and Alexandre Andreini)  in Ippon’s Richmond office during the summer of 2018, including the projects we worked on, the technologies and skills we learned, and our experiences with the unique culture at Ippon. Some of us travelled from across the Atlantic to be at Ippon and some of us travelled just down the street. Some of us were here for half a year and some of us were here for half the summer. None of our experiences were exactly the same but we all got great work experience and exposure to new technologies while having some fun at the same time.
+## The Projects
+Our time this summer was split between four main projects, COSA, Power Plant, Taiga, and an Insurance Chatbot. COSA stands for Customer Oriented Sentiment Analysis and was the project that Clement worked on in collaboration with Ippon France during the months before the rest of us started our time at Ippon. COSA is a ready to use data platform, completely based on serverless services from AWS. The goal of this particular project was to enable an end-user to react as quickly as possible to external events and organize a response; it used some AWS lambdas to collect tweets from various sources (Twitter, open data, call centers, etc.), AWS Comprehend to run data analysis, and then store the results in Elasticsearch. The Kibana plugin was then used on top of Elasticsearch to display the collected information on some dynamic dashboards.
```

Comment:
Please change the comma to a colon after "four main projects".

Reply:
Generally when introducing an acronym or initialism you first spell it out then use the initialism subsequently. For example "we worked on the Customer Oriented Sentiment Analysis (COSA) project. Blah blah blah we presented COSA to..."